zoo/auditing/standards/dummy_standards/metadata/check_randomly.yml

```diff
@@ -0,0 +1,11 @@
+namespace: constant
```

Comment:
Shouldn't the namespace be different here?

Reply:
yeowch

Reply:
changed to `random`